src/App/Header/useHeaderClassNames.ts

```diff
@@ -28,6 +25,6 @@ export default function useHeaderClassNames(headerId: string) {
   return `${
```

Comment:
Dependence on absolute positioning should be removed to allow the header to become part of the grid structure, so this whole return can be rewritten to:
```ts
  return `${
    isScrolled ? "shadow-lg" : ""
  } transition ease-in-out duration-100 z-20 w-full`;
```